Software_Carbon_Intensity/Software_Carbon_Intensity_Specification.md

```diff
@@ -123,77 +123,115 @@ The list should be maintained in alphabetic order. DELETE This Row
 
-***Embodied Emissions***
-- **Hardware Efficient Applications:** Applications that use the least physical resources possible to perform their function.
+` Operational emissions = Energy Measurement (kWh) * Carbon Intensity `
 
-## Characteristics for calculating total carbon emissions, C
+`O = E * I `
 
-### (1) `C` should be sensitive to carbon awareness, energy efficiency, or hardware efficiency improvements to the application
 
-#### Operational Emissions
 
-To calculate the operational emissions associate with software, multiply the electricity consumption of the hardware the software is running on by the regional, granular marginal emissions rate. Because this standard uses a consequential approach, marginal emissions rates should be used for electricity consumption. The marginal emissions rate reflects the change in emissions assoicated with a change in demand. 
+***Embodied Emissions (Embodied) ***
+[placeholder]
 
-Carbon aware software will optimize the timing and location of operation to minimize emissions associated with operation. This could consist of moving computation to regions with cleaner grid emissions or delaying jobs to cleaner periods (or a combination of both). Energy efficient software will also run on hardware that requires less energy to operate or the software can be re-architected to require less energy to execute. All combined, these effects should be reflected in the total operational emissions. 
+Some categories of applications that apply Embodied Emissions (Embodied): 
+- **Hardware Efficient Applications:** Applications that use the least physical resources possible to perform their function.
 
+## Characteristics for calculating total carbon emissions, `C`
 
-#### Embodied Emissions
+#### C should be sensitive to carbon awareness, energy efficiency, or hardware efficiency improvements to the application
 
-[placeholder]
 
-#### Calculating total emissions, C
+#### Calculating Total Emissions, C
 
 Total emissions, C, is the sum of operational and embodied emissions.
 
 Overall, if we calculate the total carbon emissions C of an application as X, and then we make the application carbon aware, a characteristic of the methodology of calculation is that the value of X should go down. If we make the application more energy efficient the value of X should go down. If we make the application more hardware efficient, the value of X should go down.
 
 The challenges with existing methods of calculation is that they are only sensitive to some of the above methods of making an application green. For instance there are methods of calculation of carbon emissions that are purely cost based, using those methods results in a value of C that will not not change if you make the application carbon aware or energy efficient.
 
+To calculate the carbon intensity we need to compare total emissions (C) against a baseline (R).
+
+`R` = Baseline, the other measure we will be comparing against (baseline, or carbon counterfactual) 
```

Comment:
Oh the denominator `R` previously was a unit of measure like `carbon / user`, `carbon / api-call` something like that. 

Is that what we mean here by a baseline/counterfactual? It reads to me like `carbon now / carbon before` which is a very different approach.

Reply:
@jawache This is something we certainly should discuss in more depth. @atg-abhishek is this something we can add to the agenda? I think we want to tie this to the action or choice that can be made.

Reply:
Yes for sure, the PR is the first thing we review today so let's discuss this!